src/main/java/bio/terra/common/db/DatabaseProperties.java

```diff
@@ -0,0 +1,87 @@
+package bio.terra.common.db;
+
+import org.apache.commons.dbcp2.*;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+/** Defines configuration properties for using {@link DatabaseConfiguration} use. */
+public class DatabaseProperties {
+  private boolean jmxEnabled = true;
+  private String uri;
+  private String username;
+  private String password;
+
```

Comment:
TDR has additional parameters it uses to control the size of its connection pool. I think everyone should be specifying these. Otherwise, you run the database server out of connections and get odd DB errors instead of pending on the connection pool.

See [DataRepo JdbcConfiguration](https://github.com/DataBiosphere/jade-data-repo/blob/abf95af809795b7166b431e08138094bfea39954/src/main/java/bio/terra/app/configuration/JdbcConfiguration.java#L24)